CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ X.Y.Z Release notes
 * Added `shouldCompactOnLaunch` to configuration (#507).
 * Added `Realm.compact()` for manually compacting Realm files.
 * Added various methods for permission management (#1204).
+* Added `Realm.deleteFile` for deleting a Realm (#363).
```

Comment:
We just did a release so please move this to the next version in the changelog.

Reply:
Done